includes/benchmark.py

```
@@ -22,6 +23,30 @@ def printfile(level, filename):
     filesize = os.path.getsize(filename)
     print(f"Level {level}: {filename} {filesize/1024/1024:6.1f} MiB  {filesize:12,} B")
 
+def parse_levels(level_string):
+    ''' Parse string containing comma-separated levels or level ranges '''
+    if not level_string or not level_string.strip():
+        raise ValueError("Levels string is empty")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)